pulp_python/app/tasks.py

```diff
@@ -62,6 +82,75 @@ def sync(importer_pk, repository_pk):
             changeset.apply_and_drain()
 
 
+@shared_task(base=UserFacingTask)
+def publish(publisher_pk, repository_pk):
```

Comment:
Feel free to leave as is, but I am thinking it would be nice to have tasks/sync.py and a tasks/publish.py. We don't have to, and even if we do, we don't have to do it in this PR.

Reply:
I was planning to do that right after this PR actually.